docs/articles/documentation/reference/test-api/testcontroller/expect/gte.md

```diff
@@ -8,7 +8,7 @@ permalink: /documentation/reference/test-api/testcontroller/expect/gte.html
 Asserts that `actual` is greater than or equal to `expected`.
 
 ```text
-await t.expect( actual ).gte( expected, message, options );
+await t.expect(actual).gte(expected, message, options) → this | Promise<unknown>
```

Comment:
Why we don't add `await` in action API examples? I think we should.

Reply:
@VasilyStrelyaev yay or nay?

Reply:
This is a signature, not an example, so I don't think we need `await` here